### ISSUE TITLE
cf app is not reporting the correct URL when using context path routing

### ISSUE THREAD

**Issue by crhino** [cloudfoundry/cli#809]
### Command
cf app dora

### CLI Version

cf version 6.16.1+924508c-2016-02-26

### CC API Version
2.53.0

### Details

The urls displayed on `cf app` do not include context path.

```
± ak+cp |master {1} ✓| → cf app dora
Showing health and status for app dora in org o / space s as admin...
OK

requested state: started
instances: 1/1
usage: 256M x 1 instances
urls: dora.bosh-lite.com
last uploaded: Fri Apr 1 19:17:26 UTC 2016
stack: cflinuxfs2
buildpack: ruby 1.6.14

     state     since                    cpu    memory          disk      details
#0   running   2016-04-01 12:17:43 PM   0.1%   46.2M of 256M   0 of 1G
```

```
 2016-04-01 12:22:23 ☆ ruby 2.2.4p230 lyon in ~/workspace/cf-release/src/capi-release/src/cloud_controller_ng
± ak+cp |master {1} ✓| → cf app dora --guid
4f4e6281-3f23-4ebd-b4fc-54982e7423d9

 2016-04-01 12:22:35 ☆ ruby 2.2.4p230 lyon in ~/workspace/cf-release/src/capi-release/src/cloud_controller_ng
± ak+cp |master {1} ✓| → cf curl /v2/apps/4f4e6281-3f23-4ebd-b4fc-54982e7423d9/routes
{
   "total_results": 1,
   "total_pages": 1,
   "prev_url": null,
   "next_url": null,
   "resources": [
      {
         "metadata": {
            "guid": "91ac2454-dbe8-4563-aa35-772e13fc9932",
            "url": "/v2/routes/91ac2454-dbe8-4563-aa35-772e13fc9932",
            "created_at": "2016-04-01T19:17:59Z",
            "updated_at": null
         },
         "entity": {
            "host": "dora",
            "path": "/%41",
            "domain_guid": "52a6622c-ed3f-408a-8ca0-78b15c7d1a81",
            "space_guid": "2cad6176-bd39-413b-88ef-fb4c3d86574c",
            "service_instance_guid": null,
            "port": 0,
            "domain_url": "/v2/domains/52a6622c-ed3f-408a-8ca0-78b15c7d1a81",
            "space_url": "/v2/spaces/2cad6176-bd39-413b-88ef-fb4c3d86574c",
            "apps_url": "/v2/routes/91ac2454-dbe8-4563-aa35-772e13fc9932/apps",
            "route_mappings_url": "/v2/routes/91ac2454-dbe8-4563-aa35-772e13fc9932/route_mappings"
         }
      }
   ]
}
```

**Comment by dkoper** [comment]
Thanks @crhino, either we haven't gotten around to that yet, or we simply missed it.
We'll take a look.

**Comment by dkoper** [comment]
This has been fixed in cf CLI v6.17.0.
https://github.com/cloudfoundry/cli#downloads

Thanks again!
Dies Koper
CF CLI PM
